lib/mt940_structured/file_content.rb

```diff
@@ -2,9 +2,12 @@ class MT940Structured::FileContent
   R_EOF_ING = /^-XXX$/
   R_EOF_ABN_AMRO = /^-$/
   R_EOF_TRIODOS = /^-$/
+  IBAN_LINE = /^\w{2}\d{2}\w{4}\d{6,}$/
+  BANK_NUMBER_LINE = /^\d{6,}$/
+  GIRO_NUMBER_LINE = /^P\d{6,}$/
 
-  def initialize(raw_lines, join_lines_by = ' ')
-    @raw_lines = raw_lines.map { |line| line.strip }
+  def initialize(raw_lines, join_lines_by = "\n")
+    @raw_lines = raw_lines.map(&:rstrip)
```

Comment:
I think we should avoid touching the original lines at all. That will only lead to differences with data from bank data providers.